x/wasm/keeper/msg_server_test.go

```diff
@@ -114,3 +114,160 @@ func TestMigrateExceedMaxGas(t *testing.T) {
 		NewMsgServerImpl(keeper).MigrateContract(ctx.Context(), types.NewMsgMigrateContract(creator, addr, codeID, []byte(`{"release":{}}`)))
 	})
 }
+
+func TestMsgServerStoreCode(t *testing.T) {
+	input := CreateTestInput(t)
+	ctx, accKeeper, bankKeeper, keeper := input.Ctx, input.AccKeeper, input.BankKeeper, input.WasmKeeper
+
+	deposit := sdk.NewCoins(sdk.NewInt64Coin(core.MicroLunaDenom, 100000))
+	creator := createFakeFundedAccount(ctx, accKeeper, bankKeeper, deposit)
+
+	wasmCode, err := ioutil.ReadFile("./testdata/hackatom.wasm")
+	require.NoError(t, err)
+
+	// Create contract
+	msgRes, err := NewMsgServerImpl(keeper).StoreCode(sdk.WrapSDKContext(ctx), &types.MsgStoreCode{
+		Sender:       creator.String(),
+		WASMByteCode: wasmCode,
+	})
+	require.NoError(t, err)
+	require.Equal(t, uint64(1), msgRes.CodeID)
+
+	// Verify content
+	storedCode, err := keeper.GetByteCode(ctx, msgRes.CodeID)
+	require.NoError(t, err)
+	require.Equal(t, wasmCode, storedCode)
+}
+
+func TestServerMigrateCode(t *testing.T) {
```

Comment:
TestMigrateCode

Reply:
Could i change them to `TestStoreCodeMsgServer` and `TestMigrateCodeMsgServer`? There are already `TestStoreCode` and `TestMigrateCode` in contract_test.go file